content/handbook/sales/field-operations/sales-systems/dataloader-installation.md

```
@@ -0,0 +1,83 @@
+    - 残りの番号付きステップはスキップしてください。完了です！
+1. インストール場所を見つけるために Search を使用してみてください
+    - 任意のウィンドウで、画面の右上に虫眼鏡があるはずです。それをクリックします
+    - *dataloader* と入力します
+    - 結果にフォルダがあるはずです。そのフォルダをクリックして Finder ウィンドウを表示します
+    - そのフォルダ内に、バージョン番号付きのサブフォルダが 1 つだけある場合は、「dataloader」フォルダ全体を削除できます
+    - 複数のバージョンがある場合は、不要なバージョンを削除し、保持したいバージョンを保持します。すべてを削除するには、「dataloader」フォルダ全体を削除します
+1. インストール場所が見つからなくても、あまり心配しないでください。同じ時に異なるバージョンを競合なくインストールできるはずです。または Sales Systems チームの誰かにヘルプを求めてください
+
+## バージョンのアップグレード
+
+1. [Dataloader のアンインストール](#uninstalling-dataloader)の手順に従います
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)